File: Segundo Corte/Python a Java/Square/Python/Square.py
class Square:
    def __init__(self, side):
        print("Creating square with side {s}".format(s=side))
        self.side = side
    def area(self):
        return self.side*self.side
    def perimeter(self):
        return 4*self.side
    
paint = Square(12)
table = Square(20)
window = Square(35)
print("Area: ", paint.area(),", Perimetro: ", paint.perimeter())
print("Area: ", table.area(),", Perimetro: ", table.perimeter())
print("Area: ", window.area(),", Perimetro: ", window.perimeter())

File: Segundo Corte/Python a Java/Square/Java/Square.java
public class Square {
	private double side;
	
	public Square(double side){
		System.out.println("Creating square with side: " + side);
		this.side = side;
	}
	
	public double area(){
		return this.side*this.side;
	}
	
	public double perimeter(){
		return this.side*4;
	}
}